### PR TITLE
Add/onboarding hooray component

### PR DESCRIPTION
**Pull request by bogiii**
#### Changes proposed in this Pull Request

* Introduced `Onboarding Hooray` screen

#### Testing instructions

**Hooray screen:**
- Go to `start/from/importing/wix?temp=hooray`
- Check out Hooray screen

**Progress screen:**
- Go to `start/from/importing/wix?temp=progress`
- Check out Progress screen

#### Screenshot
Screenshot 2021-11-18 at 09 18 33

Related to #57131

### CODE REVIEW

File: client/signup/steps/import-from/index.tsx [hunk 1]
```diff
@@ -10,10 +10,14 @@ import './style.scss';
 interface Props {
 	stepName: string;
 	stepSectionName: string;
+	queryObject: {
+		temp: string;
```
Comment (vishnugopal) [inline client/signup/steps/import-from/index.tsx:14]: I kind of find the name `temp` a bit weird, as it's usually the short form of `temporary`. What does it stand for here? If possible, can we find a better name? `stage` perhaps?

File: client/signup/steps/import-from/wix/index.tsx [hunk 2]
```diff
@@ -1,21 +1,39 @@
...
-				<Title>{ __( 'Importing' ) }...</Title>
-				<ProgressBar color={ 'black' } compact={ true } value={ 33 } />
-				<SubTitle>
-					{ __( "This may take a few minutes. We'll notify you by email when it's done." ) }
-				</SubTitle>
-			</Progress>
+			{ queryObject.temp === 'progress' && (
+				<Progress>
+					<Title>{ __( 'Importing' ) }...</Title>
+					<ProgressBar color={ 'black' } compact={ true } value={ 33 } />
```
Comment (vishnugopal) [inline client/signup/steps/import-from/wix/index.tsx:23]: Shouldn't `value` here also be a prop? Or are we planning to just wire this up with internal state?

**Review by vishnugopal — APPROVED** [review]
Overall looks good to me! I would prefer if `temp` is renamed though before merge!

File: client/signup/steps/import-from/wix/index.tsx (hunk 2 above)
Comment (bogiii) [reply client/signup/steps/import-from/wix/index.tsx:23]: It is temporary stuff, the temporary way to show both screens we made. The next step is to include the business logic and I'll remove the temp query param support for sure.

File: client/signup/steps/import-from/index.tsx (hunk 1 above)
Comment (bogiii) [reply client/signup/steps/import-from/index.tsx:14]: It is temporary stuff, the temporary way to show both screens we made. The next step is to include the business logic and I'll remove the temp query param support for sure.

File: client/signup/steps/import-from/wix/index.tsx (hunk 2 above)
Comment (bogiii) [reply client/signup/steps/import-from/wix/index.tsx:23]: This is a temporary state; we built the screen and want to show how to consume it. The next phase is to include business logic.

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/6991101

Thank you @bogiii for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/6991101).

**Pull request merged**